lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -870,20 +870,23 @@ def accessible_objects_for_operations(user_or_attribute, operations, options = {
       # default objects for each operation to empty list
       accessible_map = operation_names.map { |o| [o, []] }.to_h
 
-      if options[:ignore_prohibitions]
-        accessible_map.merge!(
-          objects_for_user_or_attribute_and_operations(user_or_attribute, operation_names, options)
-        )
-        return accessible_map
-      end
+      accessible_map.merge!(objects_for_user_or_attribute_and_operations(
+        user_or_attribute,
+        operation_names,
+        options
+      ))
+
+      return accessible_map if options[:ignore_prohibitions]
 
       # using `prohibitions_for` rather than just mapping to `prohibition_identifier`
       # because we want to confirm that these prohibitions exist in the db
       prohibition_names = prohibitions_for(operation_names).map { |p| operation_identifier(p) }
-      op_and_prohib_names = operation_names + prohibition_names
-      accessible_map.merge!(
-        objects_for_user_or_attribute_and_operations(user_or_attribute, op_and_prohib_names, options)
-      )
+
+      accessible_map.merge!(objects_for_user_or_attribute_and_operations(
+        user_or_attribute,
+        prohibition_names,
+        options.except(:filters)
```

Comment:
What are filters?

Reply:
they are basically `where`s that get applied to the calculation so that you can calculate, for example, only access that goes through a specific user attribute (role assignment).